### PR TITLE
Update `stylelint` from `v9.10.1` to `v13.6.1`

### PR DESCRIPTION
**Pull request by Gudahtt**
The changes made between v9.10.1 and v13.6.1 don't appear to be relevant to us, aside from bug fixes that we'd benefit from.

`gulp-stylelint` also needed to be updated, as it's in-step with `stylelint`. It went from v7 to v13.0.0. The changes aren't notable here for us either.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -61,7 +61,29 @@
...
+    debug "^4.1.0"
+    gensync "^1.0.0-beta.1"
+    json5 "^2.1.2"
+    lodash "^4.17.19"
+    resolve "^1.3.2"
+    semver "^5.4.1"
+    source-map "^0.5.0"
+
+"@babel/core@^7.4.3", "@babel/core@^7.5.5", "@babel/core@^7.7.5":
   version "7.8.3"
```
Comment (Gudahtt) [inline yarn.lock:87]: I can try to dedupe some of this

**Review by brad-decker — DISMISSED** [review]

File: yarn.lock (hunk 1 above)
Comment (whymarrh) [reply yarn.lock:87]: Don't spend too much time at this, the marginal benefit is quite low 😅

Comment (Gudahtt) [reply yarn.lock:87]: Hah, too late! Already done.

Comment (Gudahtt) [reply yarn.lock:87]: Huh, it appears that some of the changes I made had side-effects that added more duplicates. Ah well. I give up for now.

**Review by whymarrh — APPROVED** [review]
LGTM!

**Pull request merged**